src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -468,36 +466,33 @@ namespace HVACDXHeatPumpSystem {
         using EMSManager::CheckIfNodeSetPointManagedByEMS;
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int ControlNode; // control node number
-        int DXSysIndex;
-        int OutdoorAirUnitNum;    // "ONLY" for ZoneHVAC:OutdoorAirUnit
-        Real64 OAUCoilOutletTemp; // "ONLY" for zoneHVAC:OutdoorAirUnit
+        int ControlNode;                // control node number
+        Real64 OAUCoilOutletTemp = 0.0; // "ONLY" for zoneHVAC:OutdoorAirUnit
 
         int NumDXHeatPumpSystems = state.dataHVACDXHeatPumpSys->NumDXHeatPumpSystems;
-        auto &DXHeatPumpSystem(state.dataHVACDXHeatPumpSys->DXHeatPumpSystem);
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:477]:(style),[constVariable],Variable 'DXHeatPumpSystem' can be declared as reference to const